rundeckapp/grails-app/views/menu/home.next.gsp

```diff
@@ -80,7 +79,7 @@
   <div id="layoutBody">
     <div class="vue-ui-socket">
       <g:set var="createProjectAllowed"
-             value="${auth.resourceAllowedTest(action: AuthConstants.ACTION_CREATE, type: AuthConstants.TYPE_PROJECT, context: AuthConstants.CTX_APPLICATION)}"/>
+             value="${auth.resourceAllowedTest(action: [AuthConstants.ACTION_CREATE], type: AuthConstants.TYPE_PROJECT, context: AuthConstants.CTX_APPLICATION, has:true)}"/>
```

Comment:
home.next is behind a feature flag but noticed that this line wasn't 100% its equivalent in home.gsp